From fc2f20143f4e16a930c4f7ea0764e813cc6cd966 Mon Sep 17 00:00:00 2001
From: Carlos Calderon <carlos@emutex.com>
Date: Mon, 8 Oct 2018 16:06:21 +0100
Subject: [PATCH] acpi: acpi_node_add_pin_mapping added to header file.

---
 include/linux/acpi.h | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/include/linux/acpi.h b/include/linux/acpi.h
index 96d69404a54f..d2d8c1ffc03e 100644
--- a/include/linux/acpi.h
+++ b/include/linux/acpi.h
@@ -1089,6 +1089,11 @@ void __acpi_handle_debug(struct _ddebug *descriptor, acpi_handle handle, const c
 bool acpi_gpio_get_irq_resource(struct acpi_resource *ares,
 				struct acpi_resource_gpio **agpio);
 int acpi_dev_gpio_irq_get_by(struct acpi_device *adev, const char *name, int index);
+int acpi_node_add_pin_mapping(struct fwnode_handle *fwnode,
+			      const char *propname,
+			      const char *pinctl_name,
+			      unsigned int pin_offset,
+			      unsigned int npins);
 #else
 static inline bool acpi_gpio_get_irq_resource(struct acpi_resource *ares,
 					      struct acpi_resource_gpio **agpio)
@@ -1100,6 +1105,14 @@ static inline int acpi_dev_gpio_irq_get_by(struct acpi_device *adev,
 {
 	return -ENXIO;
 }
+static inline int acpi_node_add_pin_mapping(struct fwnode_handle *fwnode,
+					    const char *propname,
+					    const char *pinctl_name,
+					    unsigned int pin_offset,
+					    unsigned int npins)
+{
+	return -ENXIO;
+}
 #endif
 
 static inline int acpi_dev_gpio_irq_get(struct acpi_device *adev, int index)
-- 
2.25.1

